test/jsonInt64.spec.js

```diff
@@ -1,22 +1,69 @@
-var mocha  = require('mocha');
-var assert = require('chai').assert;
 var expect = require('chai').expect;
+var Assertion = require('chai').Assertion;
 var Int64 = require('node-int64');
 var JSONInt64 = require('../index');
 
-describe("Tests the Int64 support", function(){
+Assertion.addMethod('equalInt64', function (other)
+{
+    var object = this._obj;
+    new Assertion(object).to.be.an.instanceof(Int64);
+    new Assertion(other).to.be.an.instanceof(Int64);
+    this.assert(
+        object.equals(other),
+        "expected #{this} to be equal to #{act}",
+        "expected #{this} to be not equal to #{act}",
+        other,
+        object);
+});
+
+describe("Tests the Int64 support", function ()
+{
     var input = '{"int64MaxValue":9223372036854775807,"int64MinValue":-9223372036854775807,"number":123}';
```

Comment:
let's use the ``Number``-based constants as requested before

also this doesn't seem formatted correctly (space after punctuations)
can't this be an actual dictionary as opposed to a string?

Reply:
We can't use `Number`-based constants here.These values are way outside of `Number.MAX_SAFE_INTEGER` and `Number.MIN_SAFE_INTEGER`

This cannot be a dictionary because we are testing the JSON serialization to string and deserialization from string.

Reply:
Fixed the formatting.